src/platform/BUILD.gn

```diff
@@ -99,6 +127,13 @@ if (chip_device_platform != "none") {
     }
   }
 
+  pkg_config("glib") {
```

Comment:
If this is put here it will run for all platforms and cause an error for those that don't have glib on the build host. We should move it to the Linux directory.